pkg/restore/range.go

```diff
@@ -17,6 +19,44 @@ import (
 	"github.com/pingcap/br/pkg/rtree"
 )
 
+// Range record start and end key for localStoreDir.DB
+// so we can write it to tikv in streaming
+type Range struct {
+	Start  []byte
+	End    []byte
+	Length int
```

Comment:
Seems the `Length` field is never used, maybe we can just remove it

Reply:
Sure, it has been moved.